### PR TITLE
Sets up the initial structure for the feedback module pages

### PR DESCRIPTION
**Pull request by georgeweiler**

### CODE REVIEW

File: src/pages/Feedback/BugReport/index.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { Box } from "@chakra-ui/react"
```
Comment (r-czajkowski) [inline src/pages/Feedback/BugReport/index.tsx:2]: Let's import from our components repo.

File: src/pages/Feedback/Suggestions/index.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { Box } from "@chakra-ui/react"
```
Comment (r-czajkowski) [inline src/pages/Feedback/Suggestions/index.tsx:2]: Same as above.

File: src/pages/Feedback/UsabilitySurvey/index.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { Box } from "@chakra-ui/react"
```
Comment (r-czajkowski) [inline src/pages/Feedback/UsabilitySurvey/index.tsx:2]: Same as above.

**Review by r-czajkowski — COMMENTED** [review]
Let's fix imports and address Michał's comments and we are good to go.

File: src/pages/Feedback/index.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import BugReport from "./BugReport"
+import Settings from "./Settings"
+import { featureFlags } from "../../constants"
+
+const FeedbackPage: PageComponent = (props) => {
+  return <PageLayout title={props.title} pages={props.pages} />
+}
+
+FeedbackPage.route = {
+  path: "feedback",
```
[the remark this answers is not in the document]
Comment (georgeweiler) [reply src/pages/Feedback/index.tsx:14]: good catch -  I will investigate this bug in a future PR

**Comment by georgeweiler** [comment]
@r-czajkowski @michalsmiarowski Thanks guys, just pushed updates.

**Review by michalsmiarowski — APPROVED** [review]
LGTM

**Pull request merged**